functional-tests/src/test/groovy/FrontendCore/modules/Form_CreatePermit.groovy

```diff
@@ -0,0 +1,100 @@
+package modules
+
+import geb.Module
+import utils.Const
+
+class Form_Create_And_Edit_Permit extends Module {
+    static PERMITTEE = "Halfthor"
+    static PERMITTEE_FULL_NAME = "Halfthor Zavicus, test@blah.do"
+    static PERMIT_TYPE = "Mineral"
+    static PERMIT_STATUS = "Open"
+    static ISSUE_DATE = '1983-12-08'
+    static AMENDMENT_DATE =  "1984-12-08"
+    static MINE_NAME = "MINETEST"
+    static MODAL_HEADER = "Add a new permit to "+MINE_NAME
+
+    static at = { waitFor() {header=="Add New Contact"}}
+    static content = {
+        header {$("div", id:"rcDialogTitle0").text()}
+
+        // Permittee (wait:true) {}
```

Comment:
Does it need the wait:true {} in the comment?

Reply:
removed